package/MDAnalysis/analysis/base.py

```diff
@@ -281,25 +295,31 @@ def run(self, start=None, stop=None, step=None, verbose=None):
             stop frame of analysis
         step : int, optional
             number of frames to skip between each analysed frame
+        frames : array_like, optional
+            array of integers or booleans to slice trajectory
         verbose : bool, optional
             Turn on verbosity
+       
+
+        .. versionchanged:: 2.1.0
+            Added frames processing
```

Comment:
I changed the description to be more explicit, thank you.

Reply:
I would preferentially add the more explicit versionchanged text here rather than under `_setup_frames` since `run` is exposed in the docs but `_setup_frames` isn't.

Reply:
I added the versionchanged text to run